src/components/input/Input.tsx

```diff
@@ -0,0 +1,5 @@
+const Input = () => {
+  return <input type="text" className="" />;
+};
+
+export default Input;
```

Comment:
Since you've created a FormInput Component, what would be the use of this component.